test/runnable/sctor.d

```diff
@@ -360,7 +360,7 @@ void test14450()
 
     { auto i = immutable S14450a(1);    assert(i.x == 2); }
     { auto i = immutable S14450b(1);    assert(i.x == 2); }
-    { auto i = immutable S14450c!()(1); assert(i.x == 2); }
+    { auto i = immutable S14450c!()(1); assert(i.x == 1); }
```

Comment:
Actually, it brings it in line with the behavior a few lines above with the `== 1` code. Those cases are all non-template versions of the constructors.